src/library/interface_stereo.cpp

```diff
@@ -6,38 +6,131 @@ namespace orb_slam_2_interface {
 
 OrbSlam2InterfaceStereo::OrbSlam2InterfaceStereo(const ros::NodeHandle& nh,
                                              const ros::NodeHandle& nh_private)
-    : OrbSlam2Interface(nh, nh_private) {
+    : OrbSlam2Interface(nh, nh_private),stereo_rectified_(false) {
   // Getting data and params
   subscribeToTopics();
-  //advertiseTopics();
-  //getParametersFromRos();
+
+  if (!stereoRectification()){
+    ros::shutdown();
+    exit(1);
+  }
+
+  if(use_body_transform_){
+    if(!getBodyTransform()){
+      ros::shutdown();
+      exit(1);
+    }
+  }
+
   slam_system_ = std::shared_ptr<ORB_SLAM2::System>(
       new ORB_SLAM2::System(vocabulary_file_path_, settings_file_path_,
-                            ORB_SLAM2::System::STEREO, true));
+                            ORB_SLAM2::System::STEREO, visualization_));
 }
 
 void OrbSlam2InterfaceStereo::subscribeToTopics() {
   // Subscribing to the stereo images
   left_sub_ = std::shared_ptr<message_filters::Subscriber<sensor_msgs::Image>>(
       new message_filters::Subscriber<sensor_msgs::Image>(
-          nh_, "camera/left/image_raw", 1));
+          nh_, "cam0/image_raw", 1));
   right_sub_ = std::shared_ptr<message_filters::Subscriber<sensor_msgs::Image>>(
       new message_filters::Subscriber<sensor_msgs::Image>(
-          nh_, "camera/right/image_raw", 1));
+          nh_, "cam1/image_raw", 1));
```

Comment:
Try not to change the default topic names to make them less readable. Please remap your topics in the launchfile to what you would like them to be.

http://wiki.ros.org/roslaunch/XML/remap